lightcurve/views.py

```diff
@@ -180,5 +182,27 @@ def all_lightcurve(request):
             'images': images,
         })
 
-    return render_to_response('lightcurve_list.html', {"contexts": context_list, "request_all": True},
+    return render_to_response('lightcurve_list.html', {'contexts': context_list, 'request_all': True},
             context_instance=RequestContext(request))
+
+
+def download(request, lightcurve_id):
+    file_type = request.GET.get('file_type')
+    response = HttpResponse(content_type='text/csv')
+    response['Content-Disposition'] = 'attachment; filename="LightCurve%d.%s"' % (int(lightcurve_id), file_type) # Gets TypeError without int(). %s instead of %d?
```

Comment:
You can just format this with `%s`.  GET parameters are always string typed because they are passed in a URL string!

Reply:
Done